geoips/plugins/modules/readers/ahi_hsd.py

```diff
@@ -938,7 +938,14 @@ def sort_by_band_and_seg(metadata):
     return "{0:02d}_{1:02d}".format(band_number, segment_number)
 
 
-def call(fnames, metadata_only=False, chans=None, area_def=None, self_register=False):
+def call(
+    fnames,
+    metadata_only=False,
+    chans=None,
+    area_def=None,
+    self_register=False,
+    test_arg="AHI Default Test Arg",
```

Comment:
Added this for testing purposes, since we don't have any readers that ACTUALLY use configurable keyword arguments... Open to suggestions on how to more sustainably test this.

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/issues/334

@jsolbrig Please resolve and approve this PR when ready.  We will add unit tests at a later date.